src/taker/api.rs

```diff
@@ -193,10 +193,10 @@ impl Taker {
         data_dir: Option<PathBuf>,
         wallet_file_name: Option<String>,
         rpc_config: Option<RPCConfig>,
-        behavior: TakerBehavior,
+        #[cfg(feature = "integration-test")] behavior: TakerBehavior,
         control_port: Option<u16>,
         tor_auth_password: Option<String>,
-        connection_type: Option<ConnectionType>,
+        #[cfg(feature = "integration-test")] connection_type: Option<ConnectionType>,
```

Comment:
We can fully remove this connection type param. As everywhere it is used we can use integration-test feature flag instead. 

If integration test, do clearnet, else do tor. This will also simplify the init function.

Reply:
This should also be removed from the taker config.